compiler-rt/lib/builtins/cpu_model/x86.c

```diff
@@ -566,7 +577,7 @@ static const char *getIntelProcessorTypeAndSubtype(unsigned Family,
       *Subtype = INTEL_COREI7_SAPPHIRERAPIDS;
       break;
 
-    // Granite Rapids:
+    // Graniterapids:
```

Comment:
Looks like we prefer the invert format? https://github.com/llvm/llvm-project/pull/113881/commits/52e63a0f4b48ad106471343f2f48976bd9b84661